nodes/nodes/nodes-sno-worker-nodes.adoc

```diff
@@ -0,0 +1,69 @@
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about worker node hardware requirements, see xref:../../installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc#installation-minimum-resource-requirements_installing-restricted-networks-bare-metal[Minimum resource requirements for cluster installation].
+
+* For guidance on scaling standard clusters with a large number of worker nodes, see xref:../../scalability_and_performance/recommended-cluster-scaling-practices.adoc#recommended-scale-practices_cluster-scaling[Recommended practices for scaling the cluster].
+
+* For more information about DNS requirements for cluster nodes, see xref:../../installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc#installation-dns-user-infra_installing-bare-metal-network-customizations[User-provisioned DNS requirements].
+
+* For more information about creating a bootable ISO, see xref:../../installing/installing_sno/install-sno-installing-sno.adoc#installing-with-usb-media_install-sno-installing-sno-with-the-assisted-installer[Creating a bootable ISO image on a USB drive].
+
+* For more information about booting from an ISO using Redfish, see xref:../../installing/installing_sno/install-sno-installing-sno.adoc#install-booting-from-an-iso-over-http-redfish_install-sno-installing-sno-with-the-assisted-installer[Booting from an ISO image served over HTTP using the Redfish API].
+
+* To remove a worker node from a single-node cluster, see xref:../../nodes/nodes/nodes-nodes-working.adoc#nodes-nodes-working-deleting_nodes-nodes-working[Deleting nodes from a cluster].
+
+include::modules/sno-adding-worker-nodes-to-sno-clusters.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* During installation, you need to approve certificate signing requests for the worker node. For more information, see xref:../../nodes/nodes/nodes-sno-worker-nodes.adoc#installation-approve-csrs_add-workers[Approving the certificate signing requests for your machines].
+
+=== Adding worker nodes using the Assisted Installer API
+
+You can add worker nodes to clusters using the Assisted Installer API. Before you can add worker nodes, you need to install the OpenShift Cluster Manager CLI (`ocm`) and authenticate against the API.
+
+include::modules/cli-installing-ocm.adoc[leveloffset=+3]
+
+include::modules/ai-authenticating-against-ai-rest-api.adoc[leveloffset=+3]
+
+include::modules/ai-adding-worker-nodes-to-cluster.adoc[leveloffset=+3]
+
+[role="_additional-resources"]
+.Additional resources
+
+* During installation, you need to approve certificate signing requests for the worker node. For more information, see xref:../../nodes/nodes/nodes-sno-worker-nodes.adoc#installation-approve-csrs_add-workers[Approving the certificate signing requests for your machines].
```

Comment:
As per [ISG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#n)>> Use "need to" only when "must" is too restrictive. Suggest using **must** in this context.